internal/controllers/cluster/cluster_controller_test.go

```diff
@@ -552,10 +549,7 @@ func TestClusterReconcilerNodeRef(t *testing.T) {
 				ClusterName: "test-cluster",
 			},
 			Status: clusterv1.MachineStatus{
-				NodeRef: &corev1.ObjectReference{
-					Kind:      "Node",
-					Namespace: "test-node",
-				},
+				NodeRef: &clusterv1.MachineNodeReference{},
```

Comment:
Let's add Name: "test-node". I think that was the intention of the old code

(same above in the other case)